py/janitor/bzr_store.py

```diff
@@ -417,19 +418,20 @@ async def main_async(argv=None):
     public_runner = web.AppRunner(public_app)
     await runner.setup()
     await public_runner.setup()
+
     site = web.TCPSite(runner, args.listen_address, port=args.port)
     await site.start()
-    logging.info("Listening (private server) on %s:%s", args.listen_address, args.port)
+    logging.info("Admin API listening on %s:%s", args.host, args.port)
+
     site = web.TCPSite(public_runner, args.listen_address, port=args.public_port)
     await site.start()
-    logging.info("Listening on %s:%s", args.listen_address, args.public_port)
+    logging.info("Public website and API listening on %s:%s", args.host, args.public_port)
+
     while True:
         await asyncio.sleep(3600)
 
 
 def main():
-    import uvloop
```

Comment:
Please don't move this, especially not mixed in with other changes.

Reply:
Noted - dropped.

Was doing it as I noticed that bzr_store.py, git_store.py and site.py have slimar code structure

Reply:
REF: https://github.com/jelmer/janitor/pull/975